src/interchange/src/avro.rs

```diff
@@ -1462,6 +1462,7 @@ fn build_schema(columns: &[(ColumnName, ColumnType)], include_transaction: bool)
                 "type": "string",
                 "connect.name": "io.debezium.data.Json",
             }),
+            ScalarType::UUID => unimplemented!("uuid type"),
```

Comment:
to exercise this codepath and the other unimplemented codepath you'll need a kafka avro or avro OCF sink
(when we make a sink we generate a avro schema which is whats happening here, and we encode datums as avro values (which is whats happening in the other unimplemented block))